changelog/fragments/3221-bugfix.yaml

```diff
@@ -0,0 +1,6 @@
+entries:
+  - description: >
+      Fix bug in `operator-sdk bundle validation` that causes erroneous validation errors when the number of
+      annotations in an existing `annotations.yaml` does not equal the number of default bundle annotations
+      by upgrading the `operator-registry` dependency.
```

Comment:
Loved 👍

Reply:
Nit. The command should be `operator-sdk bundle validate`.